source/osh/datareceiver/Video.js

```diff
@@ -36,7 +36,7 @@ import DataSource from './DataSource.js';
         bufferingTime: 1000
   });
  */
-class VideoH264 extends DataSource {
+class Video extends DataSource {
```

Comment:
Update comment on [line 72](https://github.com/opensensorhub/osh-js/pull/185/commits/52ec3b77feebd03a2d549a8e9e216fddfe7d6d92#diff-e8cf31194676b08a4233b942ff52496bR72) since there is no roll value in this case

Reply:
done in 562dd5962ae9fff2f0342e2d382c6df98d43ab3b